storage/aptosdb/src/pruner/state_kv_pruner/state_kv_metadata_pruner.rs

```diff
@@ -39,14 +39,13 @@ impl StateKvMetadataPruner {
                 let mut iter = self
                     .state_kv_db
                     .db_shard(shard_id)
-                    .iter::<StaleStateValueIndexSchema>()?;
+                    .iter::<StaleStateValueIndexByKeyHashSchema>()?;
```

Comment:
Need to keep old pruner working?

Reply:
This schema should be removed before sharding migration. so safe to delete?

Reply:
no? your binary needs to run the old db same as before?

Reply:
I see.